libs/localization/README.md

```diff
@@ -189,3 +197,13 @@ Add the `extract-strings` script to `workspace.json`. Running this script will e
   }
 }
 ```
+
+Running `yarn nx run your-project:extract-strings` will extract messages from the project and create or update a Namespace entry in Contentful, if the namespace did not exist it will need to be published in Contentful before the client can query the entry.
```

Comment:
Just to follow global notation when using template within blockcode

```suggestion
Running `yarn nx run <project>:extract-strings` will extract messages from the project and create or update a Namespace entry in Contentful, if the namespace did not exist it will need to be published in Contentful before the client can query the entry.
```

Regarding "if the namespace did not exist it will need to be published in Contentful before the client can query the entry" it will be the case as well when the entry already exists in Contentful. You will need to go there and publish each time

Reply:
Hmm...

I manually created and published an empty Namespace entry in Contentful before I ran the script.
The first time I ran the script, it generated a new Contentful entry with the exact same name, and I had to delete my original entry and publish the new one.

However, I've re-run the `extract-strings` script multiple times now and it has not generated any new entries. It seems to consistently update the one it created on the first run. (I'm guessing it has something to do with authorship, etc.)

Reply:
Sorry, I did not explain myself clearly. I meant each time you extract the strings, you will need to go to Contentful and publish the entry to get the new strings through the API, not only when the entry does not exist.

Regarding the creation process, when you create an entry through Contentful it gives it a uid and you cannot manually change it through the UI, however through the extract-strings script, we give it the namespace id as uid, which is why it doesn't find the entry once you run the script again afterward